Library/Formula/cycript.rb

```diff
@@ -0,0 +1,93 @@
+class Cycript < Formula
+  desc "Modify apps using a hybrid of Objective-C++ and JavaScript syntax."
+  homepage "http://www.cycript.org"
+  url "git://git.saurik.com/cycript.git", :tag => "v0.9.592", :revision => "97bec96b43b66ab78af95d2b6c6c24f0d0a8006a"
+
+  head "git://git.saurik.com/cycript.git"
+
+  depends_on "bison" => :build
+  depends_on "android-sdk" => :build
+  depends_on "readline"
+
+  patch :DATA
```

Comment:
We usually prefere `inreplace` calls instead of a patch for Homebrew-specific stuff, so that gives:
```rb
# install support files in share instead of lib
inreplace "Execute.cpp", '"/libcycript.db"', '"/../share/libcycript.db"'
inreplace "Execute.cpp", '"/libcycript.cy"', '"/../share/libcycript.cy"'
inreplace "Java/Execute.cpp", '"/libcycript.jar"', '"/../share/libcycript.jar"'
inreplace "Makefile.am", "datdir = $(libdir)", "datdir = $(datadir)"
inreplace "Makefile.in", "datdir = $(libdir)", "datdir = $(datadir)"
```